src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Mapping/FrameworkMapping.cs

```diff
@@ -79,8 +96,26 @@ public FrameworkCodeSummary MapToFrameworkCodeSummary(FrameworkSummary framework
                 FrameworkCode = frameworkSummary.FrameworkCode,
                 Ssa1 = frameworkSummary.Ssa1,
                 Ssa2 = frameworkSummary.Ssa2,
-                Title = frameworkSummary.FrameworkName
+                Title = frameworkSummary.FrameworkName,
+                EffectiveTo = frameworkSummary.EffectiveTo
             };
         }
+
+        private bool CheckActiveFramework(string frameworkId, DateTime? effectiveFrom, DateTime? effectiveTo)
+        {
+            return DateHelper.CheckEffectiveDates(effectiveFrom, effectiveTo) || IsSpecialLapsedFramework(frameworkId);
+        }
+
+        private bool IsSpecialLapsedFramework(string frameworkId)
+        {
+            var lapsedFrameworks = _configurationSettings.FrameworksExpiredRequired;
```

Comment:
I don't understand how this knows to cast back to a list of lapsed frameworks, wouldn't var lapsedFrameworks be a string here?   Doesn't it have to be something like this:
if the config file, have:
 `<add key="FrameworksExpiredRequired" value="framework A|framework B|framework C" />`

and the line be something like:
`var lapsedFrameworks = _configurationSettings.FrameworksExpiredRequired.Split('|');`


I hope that makes sense...

Reply:
Ah I see you have some code doing something like this in
src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Settings/ApplicationSettings.cs, but separating by commas instead of pipe.  Still, I don't understand how the above works as it stands.

Reply:
In the ApplicationSettings.cs we do the same with the ElasticsearchUrls, the setting is returning a list of strings with the framework ids.

once that we have that list, we check if the framework id the we are mapping is included into that list